pages/checkIn.tsx

```diff
@@ -1,5 +1,44 @@
+import CheckInOutForm from 'components/CheckInOutForm'
+import {
+  Box,
+  Button,
+  Card,
+  CardContent,
+  CardActions,
+  Typography,
+  useMediaQuery,
+} from '@mui/material'
+
 export default function CheckInPage() {
+  const isMobileView = useMediaQuery('(max-width: 600px)')
```

Comment:
by hard-coding this value it makes it so that if we ever change the definition of what a "mobile" screen size is, we have to update the definition in multiple places. instead, use the `const theme = useTheme()` hook and call the media query like this: `const isMobileView = useMediaQuery(theme.breakpoints.down('sm');` this will make it so that it returns true whenever the viewport size is 'xs' which MUI defines as mobile